impl.hpp

```diff
@@ -481,12 +489,15 @@ void Client::on_performance(NettestFlags tid, uint8_t nflows,
     percent = (elapsed_time * 100.0 / max_runtime);
   }
   LIBNDT_EMIT_INFO("  [" << std::fixed << std::setprecision(0) << std::setw(2)
+                  << std::right << percent << "%] speed: "
+                  << format_speed_from_kbits(measured_bytes, elapsed_time));
+
+  LIBNDT_EMIT_DEBUG("  [" << std::fixed << std::setprecision(0) << std::setw(2)
                   << std::right << percent << "%]"
                   << " elapsed: " << std::fixed << std::setprecision(3)
                   << std::setw(6) << elapsed_time << " s;"
                   << " test_id: " << (int)tid << "; num_flows: " << (int)nflows
-                  << "; speed: "
-                  << format_speed_from_kbits(measured_bytes, elapsed_time));
+                  << "; measured_bytes: " << measured_bytes);
```

Comment:
I know it's suboptimal to print the message twice, but the refactoring we're planning should fix it.

(Do you mind creating an issue to briefly describe the logging refactoring?)

Reply:
Yes, the general idea here is:
- We want two different `on_performance` output formats, depending on the configured verbosity
- We don't want to start adding a bunch of `if` around to cover every possible combination of flags

The long-term solution is to have a Logger class and manage what we output by appropriately subclassing it. I'll create an issue for it.